myhpi/settings.py

```diff
@@ -57,6 +57,8 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "compressor",
+    "django_bootstrap_icons",
```

Comment:
just because I am curious: did you run the linter before committing? Or does here the insertion of dependencies in alphabetical order not apply?

Reply:
ok no I don't know whether there is more you should do

Reply:
isort and black can't understand these django dependencies, only python import statements are sorted

Reply:
In other words: Should be left like that?